pkg/apis/ceph.rook.io/v1/keys.go

```diff
@@ -21,8 +21,9 @@ import (
 )
 
 const (
-	KeyMon     rook.KeyType = "mon"
-	KeyMgr     rook.KeyType = "mgr"
-	KeyOSD     rook.KeyType = "osd"
-	KeyCleanup rook.KeyType = "cleanup"
+	KeyMon        rook.KeyType = "mon"
+	KeyMgr        rook.KeyType = "mgr"
+	KeyOSDPrepare rook.KeyType = "osdprepare"
```

Comment:
Let's be consistent with the resources that use: `ResourcesKeyPrepareOSD = "prepareosd"`. Separately, we should really merge the resources and key types so we don't have these differences.
```suggestion
	KeyOSDPrepare rook.KeyType = "prepareosd"
```